### ISSUE TITLE
Snapshot/export will fail if there is a dynamic deck that is a sub-deck of the top level standard deck.

### ISSUE THREAD

**Issue by Stvad** [Stvad/CrowdAnki#43]
Error example:
```
Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.11 (3cf770c7) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Mac 10.14.3
Flags: frz=True ao=True sv=1
Add-ons possibly involved: 1788670778

Caught exception:
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/history/archiver_vendor.py", line 39, in do_manual_snapshot
    self.do_snapshot('CrowdAnki: Manual snapshot')
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/history/archiver_vendor.py", line 47, in do_snapshot
    reason=reason)
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/history/archiver.py", line 23, in archive
    self.deck_archiver_supplier(deck).archive(reason=reason)
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/history/anki_deck_archiver.py", line 19, in archive
    deck_path = self.deck_exporter.export_to_directory(self.deck, self.output_directory)
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/export/anki_exporter.py", line 28, in export_to_directory
    deck = deck_initializer.from_collection(self.collection, deck.name)
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/representation/deck_initializer.py", line 22, in from_collection
    for child_name in direct_children]
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/representation/deck_initializer.py", line 22, in <listcomp>
    for child_name in direct_children]
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/representation/deck_initializer.py", line 13, in from_collection
    deck._load_metadata()
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/representation/deck.py", line 77, in _load_metadata
    self._load_deck_config()
  File "/Users/sitalov/Library/Application Support/Anki2/addons21/1788670778/representation/deck.py", line 81, in _load_deck_config
    new_config = DeckConfig.from_collection(self.collection, self.anki_dict["conf"])
<class 'KeyError'>: 'conf'
```

**Comment by Stvad** [comment]
see #27 for more discussion on this

**Comment by Stvad** [comment]
Fixed in v0.6.1
